packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -255,6 +256,7 @@ const Breakouts = WebexPlugin.extend({
     forEach(breakouts, (breakout: typeof Breakout) => {
       // eslint-disable-next-line no-param-reassign
       breakout.url = this.url;
+      breakout.parent = this;
```

Comment:
i feel this is like a circular dependency

Reply:
Issue: This isn't necessary. Each breakout is already in a collection and the collection's parent is the breakouts object where you store the meetingId. To access the meetingId from the breakout:
```ts
this.collection.parent.meetingId
```